api/endpoints/user.md

```diff
@@ -106,6 +106,22 @@ To update a User send JSON with updated value for one or more of the attributes.
 
 + Response 200
 
+### Edit a User's Cost Center [PUT]
```

Comment:
I think it might make more sense to add a sub-section under `Edit a User [Put]` because editing a user's cost center isn't a separate endpoint.

Also, there are 2 ways to edit a user's cost center (unfortunately). If you are assigning a user to a cost center in the default category, you use the `costCenter` property. For cost centers in any other category, you use the `costCeneters` property like you show below. Here's an example:

```
{
    "id": 45,
    "costCenter": { "id": 88 },
    "costCenters": [{"id": 172}, {"id": 2326}]
}
```

Reply:
i just happened to see this and i don't think ID is optional on our PUTs (maybe someday)

Reply:
In case you're curious, this is a relic of the past, where a user had 3 fields: `costCenter`, `costCenter1` and `costCenter2`. When the Categories feature was added, it became possible for a user to be assigned to any number of cost centers (in various categories). For some reason, we kept `costCenter` attached to the "default category" but any other cost centers now appear in the `costCenters` array (and we completely ignore `costCenter1` and `costCenter2`.